components/Header.tsx

```diff
@@ -1,95 +1,128 @@
 'use client'
 
+
 import Link from 'next/link'
 import Image from 'next/image'
 import { Popover } from '@headlessui/react'
 import { AnimatePresence, motion } from 'framer-motion'
 
-import { Logo, Button } from '@/components'
+import { Logo, Container } from '@/components'
+
+interface MobileNavLinkProps {
+  children: React.ReactNode
+  href: string
+}
+
+const MobileNavLink = ({ children, ...props }: MobileNavLinkProps) => {
+  return (
+    <Popover.Button
+      as={Link}
+      className="block text-base leading-7 tracking-tight text-gray-700"
+      {...props}
+    >
+      {children}
+    </Popover.Button>
+  )
+}
 
 const Header = () => {
+
   return (
-    <header className="absolute inset-x-0 top-0 z-50">
-      <nav
-        className="mx-auto flex max-w-7xl items-center justify-between p-6 lg:px-8"
-      >
-        <Link href="/" className="-m-1.5 p-1.5" aria-label="Home">
-          <Logo />
-        </Link>
-        <ul className="hidden items-center gap-9 lg:flex">
-          <li>
-            <Link href="/case-studies" className="-m-1.5 p-1.5 uppercase">
-              Case Studies
-            </Link>
-          </li>
-          <li>
-            <Link href="/contact" className="-m-1.5 p-1.5 uppercase">
-              Contact
+    <header className="">
```

Comment:
I can see you have a header and nav but make me harder to see your structure. You find them easy form the root page